### PR TITLE
Add scripts for doc generation

### PR DESCRIPTION
**Pull request by rickyrombo**
Refactors the `insert.js`/`swagger-import.sh` into a few files with different responsibilities:

- Added `generateExamples` which generates example responses using the example parameters and live data
- Made the existing `insert.js` into `swaggerModifiers.js` which has separate functions for each modification
- Added a new `index.js` that uses Commander to use all the different options and generate the docs using widdershins, replacing `swagger-import.sh`

Additionally:
- Updates README
- Adds basic ESLint/Prettier/Standard config (copied/adapted from Audius Protocol)
- Adds `app_name` query param to each request

NOTE: Does not include the generated examples from prod. Will do a separate PR with those to make the diff more focused.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,14 +9,28 @@
   },
   "dependencies": {},
   "devDependencies": {
+    "commander": "^9.1.0",
```
Comment (nicoback2) [inline package.json:12]: Should we pin deps? (We do so in web but doesn't seem to be the standard across repos)

File: swagger/generateExamples.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+const fetchAndSaveExample = async (urlToFetch, filename, operation) => {
+  const res = await fetch(urlToFetch)
+  let content = await res.text()
+  const contentType = getContentType(operation, res.status)
+  if (!contentType || contentType === 'application/json') {
+    try {
+      const json = JSON.parse(content)
+      console.log('Parsing as JSON', urlToFetch)
+      if (Array.isArray(json.data)) {
+        // Shorten lists
```
Comment (nicoback2) [inline swagger/generateExamples.js:39]: What's this for?

File: swagger/generateExamples.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  content = content.length > 0 ? content[0] : null
+  return content
+}
+
+const fetchAndSaveExample = async (urlToFetch, filename, operation) => {
+  const res = await fetch(urlToFetch)
+  let content = await res.text()
+  const contentType = getContentType(operation, res.status)
+  if (!contentType || contentType === 'application/json') {
+    try {
```
Comment (nicoback2) [inline swagger/generateExamples.js:35]: Nit: I like to make `try` statements as small as possible so it's clear what we think could possibly throw. Looks like that's `JSON.parse(` here?
```suggestion
    let json
    try {
        json = JSON.parse(content)
    } catch(e) {
       console.warn(...
       return
    }
    if (Array.isArray(json.data)) {
        // Shorten lists
        etc
```

File: swagger/helpers.js [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  '/users/{id}/reposts': "Get User's Reposts",
+  '/users/{id}/tags': "Get User's Most Used Track Tags",
+  '/users/{id}/connected_wallets': "Get User's Connected Wallets"
+}
+
+const convertPathToFilename = (path) => {
+  return path.substring(1).replace(/\//g, '.') + '.json'
+}
+
+module.exports = {
```
Comment (nicoback2) [inline swagger/helpers.js:97]: Can we use `export const` instead?

File: swagger/swaggerModifiers.js [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+  } catch (e) {
+    console.error('Failed to write swagger output:', e)
+  }
+}
+
+const readSwaggerFromFile = () => {
+  return require(SWAGGER_PATH)
+}
+
+module.exports = {
```
Comment (nicoback2) [inline swagger/swaggerModifiers.js:92]: ditto re: `export const`

**Review by nicoback2 — APPROVED** [review]
looks great!! 🔥

File: swagger/helpers.js (hunk 4 above)
Comment (rickyrombo) [reply swagger/helpers.js:97]: I don't think so, since I believe that's an ES Module thing but Node uses CommonJS

**Pull request merged**